components/rmrk/Gallery/ScrollTopButton.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div v-if="showBtn" class="scroll-top-button" @click="scrollToTop">
+    <b-icon icon="arrow-alt-circle-up" />
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'nuxt-property-decorator'
+import { Debounce } from 'vue-debounce-decorator'
+
+const SHOW_SCROLL_TOP_BUTTON_HEIGHT = 2000
```

Comment:
I would use `350` and move it to `~/utils/constants`

Reply:
ok